### ISSUE TITLE
what is the mAP value of the model 

### ISSUE THREAD

**Issue by abhigoku10** [chaohuangum/EECS442-final-project-complex_yolo#2]

**Comment by Hizhaoyuan** [comment]
Good performance in training set, poor test set, no generalization ability
